base_station/gui/src/components/ESTask.vue

```diff
@@ -0,0 +1,398 @@
+          <li><CommIndicator v-bind:connected="connections.lcm" name="Rover Connection Status" /></li>
+          <li><CommIndicator v-bind:connected="connections.motors && connections.lcm" name="Driving" /></li>
+        </ul>
+      </div>
+      <div class="spacer"></div>
+      <div class="help">
+        <img src="/static/help.png" alt="Help" title="Help" width="48" height="48" />
+      </div>
+      <div class="helpscreen"></div>
+      <div class="helpimages" style="display: flex; align-items: center; justify-content: space-evenly">
+        <img v-if="controlMode === 'arm'" src="/static/arm.png" alt="Robot Arm" title="Robot Arm Controls" style="width: auto; height: 70%; display: inline-block" />
+        <img v-else-if="controlMode === 'soil_ac'" src="/static/soil_ac.png" alt="Soil Acquisition" title="Soil Acquisition Controls" style="width: auto; height: 70%; display: inline-block" />
+        <img src="/static/joystick.png" alt="Joystick" title="Joystick Controls" style="width: auto; height: 70%; display: inline-block" />
+      </div>
+    </div>
+
+    <div class="box cameras light-bg">
+      <Cameras v-bind:servosData="lastServosMessage" v-bind:connections="connections.cameras"/>
+    </div>
+    <div class="box ik-controls light-bg">
+      <IKControls/>
+    </div>
+    <div class="box controls light-bg">
+      <ArmControls/>
+    </div>
+    <div class="box encoder light-bg">
+      <EncoderCounts/>
+    </div>
+    <div class="box drive light-bg">
+      <DriveControls/>
+    </div>
+    <div class="box ik-gui light-bg">
+      <iframe src="http://localhost:8020/#/" height="300" width="100%" title="IK gui"></iframe>
+    </div>
+    <div class ="box pdb light-bg">
+      <PDBFuse/>
+    </div>
+    <div class="box drive-motor light-bg">
+      <DriveVelDataV/>
+    </div>
+    <div class="spacer"></div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapMutations } from 'vuex'
+import Cameras from './Cameras.vue'
+import IKControls from './IKControls.vue'
+import CommIndicator from './CommIndicator.vue'
+import ArmControls from './ArmControls.vue'
+import DriveControls from './DriveControls.vue'
+import EncoderCounts from './EncoderCounts.vue'
+import LCMBridge from 'lcm_bridge_client/dist/bridge.js'
+import PDBFuse from './PDBFuse.vue'
+import DriveVelDataV from './DriveVelDataV.vue' 
+
+let interval;
+
+export default {
+  name: 'RATask',
+  data () {
+    return {
+      lcm_: null,
+
+      lastServosMessage: {
+        pan: 0,
+        tilt: 0
+      },
+
+      locked_joints: [],
+
+      odom: {
+        latitude_deg: 38,
+        latitude_min: 24.38226,
+        longitude_deg: -110,
+        longitude_min: -47.51724,
+        bearing_deg: 0,
+        speed: 0
+      },
+
+      connections: {
+        websocket: false,
+        lcm: false,
+        motors: false,
+        cameras: [false, false, false, false, false, false, false, false]
+      },
+
+      nav_status: {
+        completed_wps: 0,
```

Comment:
is this supposed to be here?

Reply:
No good catch, this is unnecessary for ES